doc/python-grader/index.md

```diff
@@ -0,0 +1,42 @@
+# Python Autograder
```

Comment:
Is there are reason we are using `python-grader/index.md` rather than just `python-grader.md`?

Reply:
I was planning on putting images and figures in this folder, seems you took a look at it too quickly 🙂

Reply:
haha, very good 😄